### ISSUE TITLE
Queue | Screenreader does not read success messages

### ISSUE THREAD

**Issue by araposo-tistatech** [department-of-veterans-affairs/caseflow#14272]
## Description
Screenreader does not read any success messages when completing dispatch, drafting a decision, or reviewing a decision unless they are navigated to. These alerts should always be read as soon as they pop up as they do with error alerts.

## Acceptance criteria
- [ ] Success alerts are read first

**Comment by lomky** [comment]
_[what is this chart?](https://github.com/department-of-veterans-affairs/caseflow/issues/11533)_
```bash
1 | 
2 | |||||
3 | |||||||
5 | 
8 | 
```


#### Why 2?
- Why does this work for error alerts but not success?

#### Why 3?
- Uncertainty
- Unfamilairty
